tests/e2e-leg-11/online-upgrade-session-transfer/60-kill-proxy-pod.yaml

```diff
@@ -11,8 +11,50 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: kuttl.dev/v1beta1
-kind: TestStep
-commands:
-  - command: kubectl delete deployment v-base-upgrade-pri1-sb-proxy
-    namespaced: true
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: script-kill-client-proxy
+data:
+  entrypoint.sh: |-
+    #!/bin/bash
+    set -o errexit
+    set -o xtrace
+    # get the proxy pod name
+    PROXY_LABEL=pod-template-hash,vertica.com/deployment-selector-name=v-base-upgrade-pri1-sb-proxy
+    PROXY_PRI1_ORIG=$(kubectl get pod -l $PROXY_LABEL --no-headers -o custom-columns=:.metadata.name)
+    # delete the proxy pod and a new proxy pod will be created
+    kubectl delete pod -l $PROXY_LABEL
+    PROXY_PRI1_NEW=$(kubectl get pod -l $PROXY_LABEL --no-headers -o custom-columns=:.metadata.name)
```

Comment:
What if new pod isn't ready this time($PROXY_PRI1_NEW is empty)? You can add one more step to do a simply check on the "readyReplica" of deployment. We want to know the proxy is restarted and ready.